src/EnergyPlus/DataPlantPipingSystems.hh

```diff
@@ -113,6 +115,7 @@ namespace DataPlantPipingSystems {
 
 	extern int const MeshDistribution_Uniform;
 	extern int const MeshDistribution_SymmetricGeometric;
+	extern int const MeshDistribution_Geometric;
```

Comment:
Curious.  One-sided geometric mesh distribution?  Is the compression direction predefined by cell type?